docs/getting-started/release_package.md

```diff
@@ -110,7 +114,16 @@
            + `websocket:URL`
            + `controller:node-id`
            + `edged:hostname-override`
-    
+   + Configuring docker or containerd runtime in `/etc/kubeedge/edge/conf/edge.yaml` configuration file
```

Comment:
Configuring docker or containerd runtime in `/etc/kubeedge/edge/conf/edge.yaml` configuration file  -> Configure the desired container runtime configuration in `/etc/kubeedge/edge/conf/edge.yaml` configuration file